scripts/codebuild/install.sh

```diff
@@ -20,14 +20,11 @@ echo "${magenta}----- CUSTOMIZATIONS -------${reset}"
 ####  content/*
 
 ## get environment variable from parameter store.
-export APP_CONFIG=${1}
+
+site="../../site"
+
 
 pushd scripts/gatsby-source-iiif
 yarn install
-node setupEnv.js > .env
-node getManifests.js
-node generateMD.js
-node getSchema.js
-node indexSearch.js
-node generateMDCategories.js
-popd
+
+sh ../gatsby-source-iiif/generate.sh
```

Comment:
I am not sure this will work.  Shouldn't it be?
sh ../gatsby-source-iiif/generate.sh $site 

and in generate.sh.
$1 is used twice.  Once for app config and once for site.  I think you want the 
export APP_CONFIG  
to be moved to the install.sh and generate should test to see if the env is setup similar to 
if [ -z ${APPCONFIG+x} ]; then
  print("ERROR")
  exit 1
fi